### PR TITLE
Golden pr update

### PR DESCRIPTION
**Pull request by Hekaiyo**
Removed dead links, added profile link, and fixed a test stub.

### CODE REVIEW

**Review by tonyli537 — APPROVED** [review]
Looks good. Thanks for fixing the thing.

File: features/step_definitions/user_steps.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
   when /^the updated home page with quiz results$/
     url = "https://callink.berkeley.edu/api/Organizations"
     raw_response_file1 = File.new("spec/response/raw_organizations_response.txt")
-    stub_request(:any, url).with(:query => { :page => 1, :key => Figaro.env.callink_key }).to_return(:body => raw_response_file1.read)
+    stub_request(:any, url).with(:query => { :page => 1, :key => Figaro.env.callink_key, :status => 'Active', :type => 'Registered Student Organizations' }).to_return(:body => raw_response_file1.read)
```
Comment (juanxcerv) [inline features/step_definitions/user_steps.rb:24]: good catch !

File: app/controllers/placeholder_controller.rb [hunk 2]
```diff
@@ -17,6 +17,7 @@ def home
     if not session[:logged_in]
       redirect_to landing_path
     end
+	@id = session[:user_id]
```
Comment (juanxcerv) [inline app/controllers/placeholder_controller.rb:20]: spacing is a little weird , but whatever no linter no need to fix

**Review by melvinhernandez — APPROVED** [review]
Don't see any blockers, all comments from Adnan on the Saasbook PR have been addressed! Feel free to merge 👍

**Pull request merged**